test/includes/microcloud.sh

```diff
@@ -245,9 +285,10 @@ validate_system_lxd_zfs() {
   grep -q "config: {}" <<< "${cfg}"
   grep -q "status: Created" <<< "${cfg}"
 
+  # In microcloud auto mode, the disk number is not known in advance, we just know it's the first discovered disk.
   cfg="$(lxc storage show local --target "${name}")"
   grep -q "source: local" <<< "${cfg}"
-  grep -q "volatile.initial_source: .*${local_disk}" <<< "${cfg}"
+  grep -qE "volatile.initial_source: /[^/]+(/[^/]+)*" <<< "${cfg}"
```

Comment:
This can be undone now that the disk ordering issue is fixed, but that can be in another PR.

Reply:
Fixed by #346